python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -72,13 +75,25 @@ def preprocess_ondisk_dataset(
         processed_dir_prefix, "metadata.yaml"
     )
     if os.path.exists(os.path.join(dataset_dir, preprocess_metadata_path)):
+        if force_preprocess is None:
+            with open(
+                os.path.join(dataset_dir, preprocess_metadata_path), "r"
+            ) as f:
+                preprocess_config = yaml.safe_load(f)
+            if (
+                preprocess_config.get("include_original_edge_id", None)
+                == include_original_edge_id
+            ):
+                force_preprocess = check_dataset_change(dataset_dir)
+            else:
+                force_preprocess = True
         if force_preprocess:
             shutil.rmtree(os.path.join(dataset_dir, processed_dir_prefix))
             print(
                 "The on-disk dataset is re-preprocessing, so the existing "
                 + "preprocessed dataset has been removed."
             )
-        else:
+        elif force_preprocess is False:
```

Comment:
why is the change necessary?

Reply:
It was just changed when I added the other code, do I need to change it back?